tests/src/test/scala/whisk/core/controller/test/ActivationsApiTests.scala

```diff
@@ -292,27 +306,56 @@ class ActivationsApiTests extends ControllerTestCommon with WhiskActivationsApi
         end = Instant.now)
     }.toList
     activations foreach { put(activationStore, _) }
-    waitOnView(activationStore, namespace.root, 2, WhiskActivation.collectionName)
+
+    val activationsInPackage = (1 to 2).map { i =>
+      WhiskActivation(
+        namespace,
+        EntityName(s"xyz"),
+        creds.subject,
+        ActivationId(),
+        start = Instant.now,
+        end = Instant.now,
+        annotations = Parameters("path", s"${namespace.asString}/pkg/xyz"))
+    }.toList
+    activationsInPackage foreach { put(activationStore, _) }
+
+    waitOnView(activationStore, namespace.root, 4, WhiskActivation.view)
 
     whisk.utils.retry {
       Get(s"$collectionPath?name=xyz") ~> Route.seal(routes(creds)) ~> check {
         status should be(OK)
         val response = responseAs[List[JsObject]]
-        activations.length should be(response.length)
-        activations forall { a =>
+        val allActivations = activations ++ activationsInPackage
+        allActivations.length should be(response.length)
```

Comment:
```
allActivations should have size response.length
```

Should yield a more speaking error.

Reply:
fixed these in view tests - there's a lot of these in the API tests, would rather address all of these separately as it's more than just this one suite.